lib/ae_mdw_web/controllers/aex9_controller.ex

```diff
@@ -388,13 +382,13 @@ defmodule AeMdwWeb.Aex9Controller do
     }
   end
 
-  def enc_block(:key, hash), do: :aeser_api_encoder.encode(:key_block_hash, hash)
-  def enc_block(:micro, hash), do: :aeser_api_encoder.encode(:micro_block_hash, hash)
+  defp enc_block(:key, hash), do: :aeser_api_encoder.encode(:key_block_hash, hash)
+  defp enc_block(:micro, hash), do: :aeser_api_encoder.encode(:micro_block_hash, hash)
 
-  def enc_ct(pk), do: :aeser_api_encoder.encode(:contract_pubkey, pk)
-  def enc_id(pk), do: :aeser_api_encoder.encode(:account_pubkey, pk)
+  defp enc_ct(pk), do: :aeser_api_encoder.encode(:contract_pubkey, pk)
+  defp enc_id(pk), do: :aeser_api_encoder.encode(:account_pubkey, pk)
 
-  def enc(type, pk), do: :aeser_api_encoder.encode(type, pk)
+  defp enc(type, pk), do: :aeser_api_encoder.encode(type, pk)
```

Comment:
Thank you for making all of these private, it makes the module API a lot clearer! :pray:

Reply:
It was only to validate one point. Gonna remove the aex9 branch files.